Automattic/jetpack
# Connection: Add connected user data to the initial state

Add connected user data to the initial state and consume it in the connection status card

#### Changes proposed in this Pull Request:

* Add connected user data to the initial state

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Connect site and user
* Go to the backup plugin and/or My Jetpack page
* Check the connection status card correctly displays user information and avatar
